src/Sulu/Bundle/SecurityBundle/SingleSignOn/SingleSignOnLoginRequestSubscriber.php

```diff
@@ -92,7 +92,7 @@ public function onKernelRequest(RequestEvent $event): void
 
         $adapter = $this->singleSignOnAdapterProvider->getAdapterByDomain($domain);
         if (!$adapter) {
-            if ($password) {
+            if ($password || $isResetPassword) {
```

Comment:
Looks correct as we do the same on line 83. @martinlagler can you also check if all make sense here. I needed adopt some tests so not 100% sure if I did miss something.

Reply:
@alexander-schranz Also looks good to me, approved.